src/main/resources/inspectionDescriptions/RsWhileTrueLoop.html

```diff
@@ -0,0 +1 @@
+Finds usages of the <code>while true</code> expression which can be replaced with <code>loop</code>.
```

Comment:
I think `Detects <code>while</code> loops which can be replaced with <code>loop</code>.` is a little bit better because:
* `Usage` is more about usage of some language item: function, struct, enum, etc so it can be confusing
* usually we don't use `while` loop as an expression
* in future we can make more precise condition analysis and handle more cases when `condition` is always true